test/framework/src/main/java/org/elasticsearch/test/rest/yaml/Features.java

```diff
@@ -41,6 +41,7 @@
             "headers",
             "stash_in_key",
             "stash_in_path",
+            "stash_path_magic_key",
```

Comment:
as much as I like magic, I would love to find a better name for this. stash_path_replace or something like that?